docs/api/decorators/prop.md

```diff
@@ -411,7 +410,7 @@ const AddNullToEnumModel = getModelForClass(AddNullToEnum);
 
 AddNullToEnumModel.schema.path('value').options.enum === [1, 2, null]; // true
 
-// this allows to do the following without an validation error
+// this allows you to do the following without a validation error
```

Comment:
i kinda tried not to include words like `you` in the documentation

Reply:
Ok. I'll correct differently then.

Scott